src/Umbraco.Web.UI.Client/src/views/components/blockcard/umbBlockCard.component.js

```diff
@@ -14,9 +14,38 @@
             }
         });
 
-    function BlockCardController() {
+    function BlockCardController($scope, umbRequestHelper) {
 
         var vm = this;
+        vm.styleBackgroundImage = "none";
+
+        var unwatch = $scope.$watch("vm.blockConfigModel.thumbnail", (newValue, oldValue) => {
+            if(newValue !== oldValue) {
+                vm.updateThumbnail();
+            }
+        });
+
+        vm.$onInit = function () {
+
+            vm.updateThumbnail();
+
+        }
+        vm.$onDestroy = function () {
+            unwatch();
+        }
+
+        vm.updateThumbnail = function () {
+            if (vm.blockConfigModel.thumbnail == null || vm.blockConfigModel.thumbnail === "") {
+                vm.styleBackgroundImage = "none";
+                return;
+            }
+
+            var path = umbRequestHelper.convertVirtualToAbsolutePath(vm.blockConfigModel.thumbnail);
+            if (path.toLowerCase().endsWith(".svg") === false) {
+                path += "?upscale=false&width=400)";
```

Comment:
Remove the closing ) as it generates a URL that is invalid & throws a 500 error
`http://localhost:8700/Media/3bplcy5p/18530280048_459b8b61b2_h.jpg?upscale=false&width=400)`

```suggestion
                path += "?upscale=false&width=400";
```